src/pages/ResetPassword.js

```diff
@@ -15,10 +19,14 @@ import strings from "../i18n/definitions";
 import { setTitle } from "../assorted/setTitle";
 
 export default function ResetPassword({ api }) {
-  const [email, handleEmailChange] = useFormField("");
+  const [email, setEmail, validateEmail, isEmailValid, emailErrorMsg] =
+    useFormField("", [
+      NotEmptyValidationWithMsg("Please provide an email."),
```

Comment:
Also, should this be named `NonEmptyValidation`? As a kind of validation, "non empty" seems like a more appropriate attribute?

Reply:
I have renamed all the `NotEmptyValidationWithMsg` to `NonEmptyValidation` which has the default msg of `Field cannot be empty.`